flickr.py

```diff
@@ -35,6 +35,7 @@ def search_flickr_by_tags(tags):
         "format": "json",
         "api_key": FLICKR_API_KEY,
         "tags": tags,
+        "photo_id":photo_id,
```

Comment:
This is the only change that GitHub has flagged.  I have nothing to add except that you should put a space after the colon.